src/absil/ilread.fs

```diff
@@ -3970,15 +3970,16 @@ let ClosePdbReader pdb =
 #endif
 
 let OpenILModuleReader infile opts = 
-    if IL.runningOnWindows then
```

Comment:
added https://github.com/Microsoft/visualfsharp/issues/1094 to track the kernel api vs bcl issue